onadata/libs/utils/chart_tools.py

```diff
@@ -208,6 +208,10 @@ def build_chart_data_for_field(xform, field, language_index=0, choices=None,
         field_label = 'Submission Time'
         field_xpath = '_submission_time'
         field_type = 'datetime'
+    elif isinstance(field, basestring) and field == common_tags.SUBMITTED_BY:
```

Comment:
can we remove it? if it's an object this would fail in the 2nd conjunct, why's the first part of this needed?